Assets/Scripts/Rotation.cs

```diff
@@ -0,0 +1,12 @@
+﻿using UnityEngine;
```

Comment:
Can you do this via an idle animation instead of a monobehavior?

Reply:
Done